framework/src/main/java/org/checkerframework/framework/ajava/JointJavacJavaParserVisitor.java

```diff
@@ -0,0 +1,2382 @@
+import com.sun.source.tree.LabeledStatementTree;
+import com.sun.source.tree.LambdaExpressionTree;
+import com.sun.source.tree.LiteralTree;
+import com.sun.source.tree.MemberReferenceTree;
+import com.sun.source.tree.MemberSelectTree;
+import com.sun.source.tree.MethodInvocationTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.ModifiersTree;
+import com.sun.source.tree.ModuleTree;
+import com.sun.source.tree.NewArrayTree;
+import com.sun.source.tree.NewClassTree;
+import com.sun.source.tree.OpensTree;
+import com.sun.source.tree.PackageTree;
+import com.sun.source.tree.ParameterizedTypeTree;
+import com.sun.source.tree.ParenthesizedTree;
+import com.sun.source.tree.PrimitiveTypeTree;
+import com.sun.source.tree.ProvidesTree;
+import com.sun.source.tree.RequiresTree;
+import com.sun.source.tree.ReturnTree;
+import com.sun.source.tree.StatementTree;
+import com.sun.source.tree.SwitchTree;
+import com.sun.source.tree.SynchronizedTree;
+import com.sun.source.tree.ThrowTree;
+import com.sun.source.tree.Tree;
+import com.sun.source.tree.Tree.Kind;
+import com.sun.source.tree.TreeVisitor;
+import com.sun.source.tree.TryTree;
+import com.sun.source.tree.TypeCastTree;
+import com.sun.source.tree.TypeParameterTree;
+import com.sun.source.tree.UnaryTree;
+import com.sun.source.tree.UnionTypeTree;
+import com.sun.source.tree.UsesTree;
+import com.sun.source.tree.VariableTree;
+import com.sun.source.tree.WhileLoopTree;
+import com.sun.source.tree.WildcardTree;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import org.checkerframework.javacutil.BugInCF;
+
+/**
+ * A visitor that processes javac trees and JavaParser nodes simultaneously, matching corresponding
+ * nodes.
+ *
+ * <p>By default, visits all children of a javac tree along with corresponding JavaParser nodes. The
+ * JavaParser node corresponding to a javac tree is always passed as the secondary parameter to the
+ * {@code visit} methods.
+ *
+ * <p>To perform an action on a particular tree type, override one of the methods starting with
+ * "process". For each javac tree type JavacType, and for each possible JavaParser node type
+ * JavaParserNode it may be matched to this class contains a method {@code
+ * processJavacType(JavacTypeTree javacTree, JavaParserNode javaParserNode)}. These are named after
+ * the visit methods in {@code com.sun.source.tree.TreeVisitor}, but for each javac tree type there
+ * may be multiple process methods for each possible node type it could be matched to.
+ *
+ * <p>The {@code process} methods are called in pre-order. That is, process methods for a parent are
+ * called before its children.
+ */
+public abstract class JointJavacJavaParserVisitor implements TreeVisitor<Void, Node> {
+    @Override
+    public Void visitAnnotation(AnnotationTree javacTree, Node javaParserNode) {
+        // javac stores annotation arguments as assignments, so @MyAnno("myArg") might be stored the
```

Comment:
Should "might be" be "is"?  If not, under what circumstance is the statement about storage true?